.github/actions/releasebot/action.yml

```
@@ -0,0 +1,250 @@
+
+            const packages = releases.map(({ name, version }) => {
+              const pkgFile = path.join(packageRoot, pkg);
+              const content = JSON.parse(await readFile(pkgFile, 'utf8'));
+              packageMap.set(content.name, `./packages/${pkg.replace('/package.json', '')}`);
+            });
+          }
+
+          await generatePackageMap();
+
+          return JSON.stringify(Object.fromEntries(packageMap));
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)